### PR TITLE
docs: add the description of the default slot to generate better code hint files

### LINKED ISSUE #2891: [Bug Report] Some tags are not recognized by webstorm
### Element Plus version
1.0.2-beta.70

### OS/Browsers version
windows/chrome

### Vue version
3.1.5

### Reproduction Link
https://jsfiddle.net/iamkun/ods20k7t/as

### Steps to reproduce
Only found ``` <el-main>``` for the time being 
There may be others

### What is Expected?
Hope to fix it. Starting from element-ui2, it is all ide friendly. I hope it can be maintained, thanks

### What is actually happening?
some tags are not recognized by webstorm

fff

### PR DESCRIPTION
**Pull request by tolking**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

close #2891 

Used to generate better code hint files

- Add the description of the default slot
- optimize some documents to generate better code hint files

### CODE REVIEW

File: website/docs/en-US/container.md [hunk 1]
```diff
@@ -252,17 +252,46 @@ These components use flex for layout, so please make sure your browser supports
...
+
+| Name | Description |
+| --- | -------- |
+| — | customize footer content |
+
+### Main Slots
+
+| Name | Description |
+| ---- | -------- |
+| — | customize main content |
```
[the remark this answers is not in the document]
Comment (tolking) [reply website/docs/en-US/container.md:297]: The lack of a description of the Main in the document caused the inability to generate relevant code hint from the document.

Adding Slots only optimizes the generated `web-types.json `.

I am confused that some components in the document have but some components do not have a description of the default slot. I need some advice

[the remark this answers is not in the document]
Comment (OneSeven) [reply website/docs/en-US/container.md:297]: > I think it is unnecessary.

Really use vscode per capita? Now, element-plus is very rough, and many documents are missing and incomplete. Please don’t stop the behavior that makes element-plus better! ! ! ! !！！！！！！！！！

[the remark this answers is not in the document]
Comment (kooriookami) [reply website/docs/en-US/container.md:297]: OK, I ignored `web-types.json` before. It's nice that you could add all languages document.

[the remark this answers is not in the document]
Comment (tolking) [reply website/docs/en-US/container.md:297]: Next I will refine all the default slots

**Comment by tolking** [comment]
> @tolking Hi, Thanks for your contribution.
> Your commits includes some irrelevant ones. Could you rebase your commits please? Thanks.

I don't understand which part you are referring to

**Review by sxzz — APPROVED** [review]

**Comment by tolking** [comment]
Due to the change of document structure. I decided to create a new PR to replace this

**Pull request closed without merging**